website/components/io-home-card/index.tsx

```diff
@@ -0,0 +1,119 @@
+import * as React from 'react'
+import Link from 'next/link'
+import InlineSvg from '@hashicorp/react-inline-svg'
+import camelCase from 'camelcase'
+import classNames from 'classnames'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import { IconExternalLink16 } from '@hashicorp/flight-icons/svg-react/external-link-16'
+import { productLogos } from './product-logos'
+import s from './style.module.css'
+
+interface IoHomeCardProps {
+  variant?: 'light' | 'gray' | 'dark'
+  products?: Array<keyof typeof productLogos>
+  link: {
+    url: string
+    type: 'inbound' | 'outbound'
+  }
+  inset?: 'none' | 'sm' | 'md'
+  eyebrow?: string
+  heading?: string
+  description?: string
+  children?: React.ReactNode
+}
+
+function IoHomeCard({
+  variant = 'light',
+  products,
+  link,
+  inset = 'md',
+  eyebrow,
+  heading,
+  description,
+  children,
+}: IoHomeCardProps) {
+  const space = camelCase(['space', inset])
+
+  const LinkWrapper = ({ className, children }) =>
+    link.type === 'inbound' ? (
+      <Link href={link.url}>
+        <a className={className}>{children}</a>
+      </Link>
+    ) : (
+      <a
+        className={className}
+        href={link.url}
+        target="_blank"
```

Comment:
Question: Do we always want a new tab?

Reply:
That is a great question, from what I have seen elsewhere that was the typical pattern.

Reply:
For sure, funny enough I just stumbled on another case where this is the default for `outbound`. Maybe worth it to discuss default UX like this as a group and have alignment.

Reply:
Notice our button component takes this stance too.
> If true, rel="noopener" and target="_blank" will be set.